### ISSUE TITLE
Rack 1.0 doesn't execute (Probably due to non-english characters in documents path)

### ISSUE THREAD

**Issue by Mogur8** [VCVRack/Rack#1377]
Since I downloaded v1.0, Rack.exe doesn't execute. I've been talking with modlfo (Vult) and he suggested that the problem may be caused by my username having non-english characters (Gal**á**n).
I've installed Rack in the root folder of the hard drive (D:\VCV\Rack) but this doesn't seem to work.
Is there a way to workaround this?

### Details

Operating system: Windows 10
Rack version: v1.0 (and 0.6, it used to work properly, but now it doesn't)
Hardware relevant to your issue: None
Log file: There is not

**Comment by Coirt** [comment]
#436

Can you change your username?

**Comment by Mogur8** [comment]
I can change my username, but that doesn't rename the folder files. I can do it manually, but that seems a bit risky, It's easy to break paths (and I'm not much of a computer savy)

**Comment by Coirt** [comment]
There is a workaround but it requires editing the registry >
https://support.microsoft.com/en-us/help/2454362/renaming-a-user-account-does-not-automatically-change-the-profile-path

a little more detailed >
https://superuser.com/questions/890812/how-to-rename-the-user-folder-in-windows-10

**Comment by Mogur8** [comment]
Thank you very much, I'll take a look at those when I have a little more time. Anyways, I was planning to reinstall my operating system, so this may be my cue to do so.
However, I believe this should be solved inside Rack's code, right? I don't mean to be demanding, I'm absolutely grateful for the effort that Andrew and everyone else is putting into this project. But this may cause some people to never try this piece of art. Besides that, v0.6 was working nicely, so are the rest of my software programs: Something must have changed between both versions that cause this disruption.

**Comment by Coirt** [comment]
No problem!

**Comment by AndrewBelt** [comment]
This was already solved in Rack 1.0. Without debugging, I can't guess what the problem is (crystal ball too hazy).

**Comment by Mogur8** [comment]
How can I find out what is happening? I would gladly help debugging, but I don't really know what to do.
It's really strange, I've been using Rack since v0.3 and this has not happened before. 
I'll try to change my username following Coirt's guides this afternoon and report back.

**Comment by Mogur8** [comment]
A friend recommended me to add `-d` to the shortcut and I was able to open Rack by using the command `Rack.exe -d` . However, this opens Rack in "development mode", so it can't connect to the library, and I only get the Core plugins, not my purchased VCV products (or third parties)

**Comment by AndrewBelt** [comment]
Development mode isn't what you want, probably `-u`. See https://vcvrack.com/manual/Installing.html#command-line-usage
No idea why you're having the issue though. I've added debugging in 382f67a which should report to the terminal when the log.txt file can't be created. If you compile from source (or wait and use Rack 1.1.0), launching Rack from a terminal will probably give the answer.

**Comment by Mogur8** [comment]
`-u` doesn't work (nor `-u <dir>` I tried with `-s` too. I tried launching from a terminal too, but that didn't print any log message or anything. Maybe in this new branch, it'll work. Thank you!
I can compile from source (by following Leonardo's instructions, I don't really know how to do that on my own), but I think I'll reinstall my OS first. My machine needs it, anyway. 
I'll report when I can find a couple of hours to do that.
